Patch01 gwen mb

gwenMB opened the pull request:
I found a number of issues in the Gutenberg transcription of The Blue Castle by comparing it to the Wikisource transcription.  Since the Gutenberg transcription was used as a source for the SE edition, many of the problems carried over.  Please review my changes and decide if you would like to pull them into the SE ebook.

Thanks,
Gwen

gwenMB committed:
Formatting - missing paragraph breaks

“Well, yes,” he said finally. “There were some women I might have

“So I see,” said Mrs. Frederick. “May I ask why?”

Mistawis was drowned in sunset lilac light

gwenMB committed:
Formatting - missing italics

poor Valancy was—a little—_you_ understand?”

Cissy if _you_ would behave yourself,”

She would _not_ be littered with them.

Wear _that_ to a party!

“I will _not_ believe,” said Cousin Stickles solemnly

I’ve been wanting to see you _very_

The rest are dead. He _isn’t_

gwenMB committed:
Formatting - move italics around "Go hang!"

The "Go" in "Go hang!" is also italicized in the text, although it is
hard to see.  Compare the capital g in "Go" to the one in "Cissy Gay"
on the facing page.

gwenMB committed:
Formatting - move italics around ‘None!’

‘None!’ is a complete quoted sentence, I belive that punctuation should be included in the italics in this case.

gwenMB committed:
Formatting - move italics around ‘Oh, darn!’

The word "Oh" should be italicized, as per page scans. Also, ‘Oh, darn!’ is a comple quoted sentence, I belive that punctuation should be included in the italics in this case.

gwenMB committed:
Reinstate duplicate word, *that that* Snaith scoundrel

Match page scans. The sentence is gramatically correct as shown.

"The worst feature of the case is that that Snaith
scoundrel is always hanging around Roaring Abel's,"

All editions at IA (13 of 13) keep "that that"
https://archive.org/search?query=%28%22The+worst+feature+%22%29+AND+title%3A%28the+blue+castle%29+AND+mediatype%3A%28texts%29&sin=TXT&sort=-date

gwenMB committed:
Reinstate duplicate word, *that that* jail-bird, Snaith

Match page scans. The sentence is gramatically correct as shown.

"This is no fit place for you to be, and
you know it. Why, I'm told that that jail-bird,
Snaith, is hanging around here every evening."

All editions at IA (13 of 13) keep "that that"
https://archive.org/search?query=%28%22that+jail-bird%22+OR+%22that+jailbird%22%29+AND+title%3A%28the+blue+castle%29+AND+mediatype%3A%28texts%29&sin=TXT&sort=-date

gwenMB committed:
Reinstate duplicate word, And I tell *you you*

Match page scans. The sentence is gramatically correct as shown.

“I can express my feelings without blasphemy.
And I tell you you have covered yourself with
eternal disgrace and infamy by marrying that
drunkard——”

All editions at IA (13 of 13) keep "you you"
https://archive.org/search?query=%28%22And+I+tell+you%22%29+AND+title%3A%28the+blue+castle%29+AND+mediatype%3A%28texts%29&sin=TXT&sort=-date

gwenMB committed:
Reinstate duplicate word, *her her* ailment

Match page scans. The sentence is gramatically correct as shown.

Dr. Trent was as gruff and abrupt as usual, but
he did not tell her her ailment was imaginary.

All editions at IA (13 of 13) keep "her her"
https://archive.org/search?query=%28%22her+ailment%22%29+AND+title%3A%28the+blue+castle%29+AND+mediatype%3A%28texts%29&sin=TXT&sort=-date

gwenMB committed:
Typo, punctuation "huge enjoyment."

Match the page scans. Note that there is cruft on the page in this location.

All other editions of the book on Internet archive have a period and not an exclamation point here.

https://archive.org/search?query=%28%22huge+enjoyment%22%29+AND+title%3A%28the+blue+castle%29+AND+mediatype%3A%28texts%29&sin=TXT&sort=-date

gwenMB committed:
Typo, punctuation hush-hushed

Match the page scans.

gwenMB committed:
Typo, punctuation "Now she almost counted"

Match the page scans. They do show cruft on the page that is easy to mistake for a comma.

However, the 1926 A. L. Burt Company edition scan clearly shows no punctuation here.

https://archive.org/details/bluecastle0000lucy/page/234/mode/2up?q=%22she+almost+counted%22&view=theater

gwenMB committed:
Typo, punctuation missing apostrophe “I—can’t.”

Match page scans.

gwenMB committed:
Typo, punctuation "she muttered,"

Page scan shows comma, transcription shows period.

gwenMB committed:
Typo, punctuation "said Valancy thoughtfully"

Match page scans.

gwenMB committed:
Typo, function --> functions

Match the page scans. It is somewhat difficult to see, but it is there.

gwenMB committed:
Typo, intervals --> intervales

Page scan shows "intervales." Editors probably thought they were correcting a typo in the original, but "intervales" is a real word that  means bottomland, or a low-lying land along a watercourse. The word is relevant and makes sense in the context here.

https://www.merriam-webster.com/dictionary/intervales

Also, most other editions show "intervales", although a couple of newish editions made the same change to "intervals"

https://archive.org/search?query=%28%22places+and+silent%22%29+AND+title%3A%28the+blue+castle%29+AND+mediatype%3A%28texts%29&sin=TXT&sort=-date

gwenMB committed:
Add language semantics.

These were italicized in the text as words borrowed from the french language, but are now found in the english M-W dictionary.
* à la
* cachet
* en route

gwenMB committed:
[Editorial] typo, insignificent --> insignificant

Suspected publisher typo.

Most editions make this correction (11 of 13)

https://archive.org/search?query=%28%22an+insignificent%22+OR+%22an+insignificant%22%29+AND+title%3A%28the+blue+castle%29+AND+mediatype%3A%28texts%29&sin=TXT&sort=-date

gwenMB committed:
Semantics for graphemes

gwenMB committed:
Update word count in metadata

gwenMB committed:
[Editorial] typo, dose --> doze

Suspected publisher typo.

Most editions do not make this correction (2 of 13), however it can be seen here:

https://archive.org/details/bluecastle0000lucy/page/254/mode/1up?view=theater&q=doze

Other uses of the word "dose" are referring to medicine:

* She was to get the prescription he enclosed filled and carry it with her always, taking a dose whenever her attacks came on

* What she wants is a generous dose of spank-weed and I’d like to be the one to administer it.

* She was so bad at first that she could not even get herself a dose of Dr. Trent’s prescription.

But the use that I am changing to "doze" is talking about falling asleep:

* Towards morning Valancy fell into a fitful doze with ridiculous dreams.

M-W definitions for "dose" and "doze":

https://www.merriam-webster.com/dictionary/dose

https://www.merriam-webster.com/dictionary/doze

gwenMB committed:
[Editorial] typo, stay --> stray

Suspected publisher typo.

Most editions make this correction (12 of 13)

https://archive.org/search?query=%28%22glinted%22%29+AND+title%3A%28the+blue+castle%29+AND+mediatype%3A%28texts%29&sin=TXT&sort=-date

gwenMB committed:
Semantics for internal thoughts

gwenMB committed:
Reverting comma that was added to make lint happy.

acabal commented on src/epub/text/chapter-19.xhtml:
```diff
@@ -53,7 +53,7 @@
 			<p>“She’s giving in,” thought Roaring Abel. “She’ll go with him. Beats all, the power these preacher fellows have over women.”</p>
 			<p>Valancy <em>was</em> on the point of obeying <abbr epub:type="z3998:name-title">Dr.</abbr> Stalling. She must go home with him⁠—and give up. She would lapse back to Doss Stirling again and for her few remaining days or weeks be the cowed, futile creature she had always been. It was her fate⁠—typified by that relentless, uplifted forefinger. She could no more escape from it than Roaring Abel ...
-			<p>“<em>Fear is the original sin</em>,” suddenly said a still, small voice away back⁠—back⁠—back of Valancy’s consciousness. “<em>Almost all the evil in the world has its origin in the fact that someone is afraid of something.</em>”</p>
+			<p>“<q>Fear is the original sin</q>,” suddenly said a still, small voice away back⁠—back⁠—back of Valancy’s consciousness. “<q>Almost all the evil in the world has its origin in the fact that someone is afraid of something.</q>”</p>
```
This should remain `<em>`. `<q>` is used for direct quotations without quotation marks. (Browsers by default render `<q>` with quotation marks and not italics.) If this character is actually quoting something from earlier in the book, then this should be nested single quotes and not italics.

gwenMB replied:
This was a quote from a fictional book, so I did an editorial commit d0bbc302e0c4250d072da5ef3da930ece52dae2c

acabal commented on src/epub/text/chapter-32.xhtml:
```diff
@@ -10,10 +10,10 @@
 			<p>New Year. The old, shabby, inglorious outlived calendar came down. The new one went up. January was a month of storms. It snowed for three weeks on end. The thermometer went miles below zero and stayed there. But, as Barney and Valancy pointed out to each other, there were no mosquitoes. And the roar and crackle of their big fire drowned the howls of the north wind. Good Luck and Banjo waxe ...
 			<p>“But they’ll come back in spring,” promised Barney.</p>
-			<p>There was no monotony. Sometimes they had dramatic little private spats that never even thought of becoming quarrels. Sometimes Roaring Abel dropped in⁠—for an evening or a whole day⁠—with his old tartan cap and his long red beard coated with snow. He generally brought his fiddle and played for them, to the delight of all except Banjo, who would go temporarily insane and retreat under Valan ...
+			<p>There was no monotony. Sometimes they had dramatic little private spats that never even thought of becoming quarrels. Sometimes Roaring Abel dropped in⁠—for an evening or a whole day⁠—with his old tartan cap and his long red beard coated with snow. He generally brought his fiddle and played for them, to the delight of all except Banjo, who would go temporarily insane and retreat under Valan ...
```
`<span>` is not necessary, `a la` is English.

gwenMB replied:
Removed in commit 6b6fe53c8e1f9b63985b9a1a8109eddf852d4099

acabal commented on src/epub/text/chapter-34.xhtml:
```diff
@@ -9,7 +9,7 @@
 			<h2 epub:type="ordinal z3998:roman">XXXIV</h2>
 			<p>Valancy had two wonderful moments that spring.</p>
-			<p>One day, coming home through the woods, with her arms full of trailing arbutus and creeping spruce, she met a man who she knew must be Allan Tierney. Allan Tierney, the celebrated painter of beautiful women. He lived in New York in winter, but he owned an island cottage at the northern end of Mistawis to which he always came the minute the ice was out of the lake. He was reputed to be a lon ...
+			<p>One day, coming home through the woods, with her arms full of trailing arbutus and creeping spruce, she met a man who she knew must be Allan Tierney. Allan Tierney, the celebrated painter of beautiful women. He lived in New York in winter, but he owned an island cottage at the northern end of Mistawis to which he always came the minute the ice was out of the lake. He was reputed to be a lon ...
```
`<span>` is not necessary, `cachet` is English. Removing italics is enough.

gwenMB replied:
Removed in commit 6b6fe53c8e1f9b63985b9a1a8109eddf852d4099

acabal reviewed (changes requested):
Great work. Some minor tweaks to be made and I can merge this in. Thanks!

gwenMB committed:
Partial revert -  language semantics.

Per Alex Cabal,
<span> is not necessary, a la is English.
<span> is not necessary, cachet is English. Removing italics is enough.

gwenMB committed:
[Editorial] Valancy's internal thoughts are quotes from book.

Removing italics, replacing with single quotes inside double quotes.

Valancy is quoting from John Foster's book, Magic of Wings.
Previously, she read these lines at the end of Chapter 5 (page 36)
https://babel.hathitrust.org/cgi/pt?id=uva.x000134270&seq=48

“<em>Fear is the original sin</em>,” wrote John Foster. “<em>Almost all the evil in the world has its origin in the fact that someone is afraid of something.</em> It is a cold, slimy serpent coiling about you. It is horrible to live with fear; and it is of all things degrading.”

Later in chapter 19 (page 142), when she is about to give in to her fear, she remembers the quoted lines and it helps her gather courage to advocate for herself.
https://babel.hathitrust.org/cgi/pt?id=uva.x000134270&seq=154

“ ‘Fear is the original sin,’ ” suddenly said a still, small voice away back⁠—back⁠—back of Valancy’s consciousness. “ ‘Almost all the evil in the world has its origin in the fact that someone is afraid of something.’ ”

Per Alex Cabal:
If this character is actually quoting something from earlier in the book, then this should be nested single quotes and not italics.

acabal commented:
Great, thanks!
